docs/reST/ref/draw.rst

```diff
@@ -163,6 +166,9 @@ object around the draw calls (see :func:`pygame.Surface.lock` and
          .. note::
             When using ``width`` values ``> 1``, the edge lines will only grow
             inward.
+   :param int quadrant: (optional) used for drawing only one quadrant of the circle
+      Has to be in the interval [0, 4]. If ``quadrant == 0`` entire circle will be 
+      drawn
```

Comment:
Is 0-4 confusing here? I'm not sure if it's better than 0-3, and a default None for quadrant.

If kwargs were used (top_left, top_right, bottom_left, bottom_right) each defaulting to True, then it would make it possible from the API that different quadrants could be True and drawn at once. I'm not sure if that's desirable?